core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -515,6 +519,8 @@
   GC_CYCLE_DELAY("gc.cycle.delay", "5m", PropertyType.TIMEDURATION,
       "Time between garbage collection cycles. In each cycle, old RFiles or write-ahead logs "
           + "no longer in use are removed from the filesystem."),
+  GC_HOST("gc.bind.address", "0.0.0.0", PropertyType.HOSTLIST,
```

Comment:
Used PropertyType.HOSTLIST because it already did some validation even through these would only be a single entry.

Reply:
Probably okay for a first pass, but probably better to create a new type. These need to specifically be IP addresses (IPv4 or IPv6) and *not* hostnames. They would also be a single entry (and might permit a port to be specified... possibly).

Reply:
The HOSTLIST validation includes checking for port which is why I thought to go ahead and use it but did not realize that ip addresses were required.  A new type would be a good idea.